core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navbar-breadcrumb.directive.html

```diff
@@ -8,11 +8,11 @@
         <[getCurrentTabName()]>
       </span>
     </span>
-    <span ng-if="navbarTitle === ''" ng-click="editTitle()" style="opacity: 0.6;">
-      Untitled Exploration
-    </span>
     <span ng-if="navbarTitle === null">
       Loading...
     </span>
+    <span ng-if="navbarTitle === ''" ng-click="editTitle()" style="opacity: 0.6;">
+      Untitled Exploration
+    </span>
```

Comment:
The issue was both `Untitled exploration Loading...` were appearing at the same time, will these changes fix the issue? Can you share a video/screenshot of the new behavior?

Reply:
Yes this issue can be solved with this small tweek.

Reply:
@rohitkatlaa I was thinking about the reason for the issue and I cannot find out through this fix, do you know the reason why it's happening in the first place? 

Also, does it also fixes the issue mentioned in [this comment](https://github.com/oppia/oppia/issues/7543#issuecomment-527267341)?

Reply:
The issue that I fixed was https://github.com/oppia/oppia/issues/7543#issue-488346797 .
I am sorry but I am not sure as to what is the issue and what is to be expected in https://github.com/oppia/oppia/issues/7543#issuecomment-527267341

Reply:
Hi @rohitkatlaa, Sorry for the late reply, In general, if you want me to review , reply or need suggestion please add me to the assignee list that way I'll be able to reply you quickly! 

  > The issue that I fixed was #7543 (comment) .

If you are going to fix a part of the issue then please mention this in your PR title.

> I am sorry but I am not sure as to what is the issue and what is to be expected in #7543 (comment)

Sorry for not writing the expected behavior, I've added it now PTAL!

Can you also reply to the earlier comment: 
> @rohitkatlaa I was thinking about the reason for the issue and I cannot find out through this fix, do you know the reason why it's happening in the first place?

Reply:
I guess what is happening is that since "loading" is after "untitled exploration", "untitled exploration" renders first and then loading is hidden.